plugins/AtlasExplorer/api/IUseCase.h

```diff
@@ -23,17 +23,31 @@
 #include <brayns/engine/model/Model.h>
 #include <brayns/json/JsonType.h>
 
-#include <api/AtlasVolume.h>
+#include "Atlas.h"
 
 class IUseCase
 {
 public:
     virtual ~IUseCase() = default;
 
+    /**
+     * @brief Returns the identification name of the use case.
+     * @return std::string
+     */
     virtual std::string getName() const = 0;
 
-    virtual bool isVolumeValid(const AtlasVolume &volume) const = 0;
+    /**
+     * @brief Tests wether the use case can be applied to the atlas.
+     * @param atlas Atlas to be tested.
+     * @return true If the atlas is valid for this use case, false otherwise.
+     */
+    virtual bool isAtlasValid(const Atlas &atlas) const = 0;
```

Comment:
isValidAtlas ?

Reply:
For me it goes on how would you ask the question:
Is the atlas valid?
or
Is valid the atlas?
To me it makes more sense isAtlasValid